helm/aws-operator/values.yaml

```diff
@@ -34,10 +31,10 @@ project:
 #
 resource:
   default:
-    name: '{{ .Release.Name | replace "." "-" | trunc 47 }}'
+    name: '{{ include "aws-operator.resourceName" . }}'
     namespace: "giantswarm"
   psp:
-    name: '{{ .Release.Name | replace "." "-" | trunc 47 }}-psp'
+    name: '{{ include "aws-operator.resourceName" . }}-psp'
   pullSecret:
-    name: '{{ .Release.Name | replace "." "-" | trunc 47 }}-pull-secret'
+    name: '{{ include "aws-operator.resourceName" . }}-pull-secret'
     namespace: "giantswarm"
```

Comment:
Wouldn't it be better to move all of that to the helper? I didn't want to make it configurable from the outside. I think it makes little sense. Back then it was a compromise between having it here and repeating this everywhere in the resources, because I didn't know about _helpers.tpl pattern.

Reply:
I'm not sure I follow, we can't move the entire `resource` block to the helper because it isn't used as-is, only individual variables are used in other templates.

I suppose we could define variables like `aws-operator.resourceName`, `aws-operator.pspResourceName` and `aws-operator.pullSecretResourceName` in the helper instead and remove the `resource` block from `values.yaml`, but what about the namespace? Shouldn't it be configurable?

Reply:
I thought defining variables like `resource.default.name` etc. Do you think `aws-operator` prefix is important? I don't think the namespace should be configurable. We don't have the usecase for that and we don't even know if it would work if we changed the namespace for a single resource.

Reply:
Taking over here.

* moved the entire `resource` block from values to helpers
* changed from `tpl` to `include` in actual templates
* replaced pullSecret namespace with default namespace, because well they are just the same